### ISSUE TITLE
Unable to get list of available templates

### ISSUE THREAD

**Issue by gongled** [essentialkaos/terrafarm#17]
I've found that in case of renaming template directories I can't get list of available templates via command `terrafarm templates`. I think it may be unpredictable. 

**Steps to reproduce**
1. Get the latest version of Terrafarm
2. Switch to the templates directory
3. Rename `c6-multiarch` template
4. Run `terrafarm templates`.

**Environment**
```
$ uname -ar
Linux hostname 3.10.0-327.10.1.el7.x86_64 #1 SMP Tue Feb 16 17:03:50 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

$ cat /etc/redhat-release
CentOS Linux release 7.2.1511 (Core)

$ terrafarm -v

Terrafarm 0.7.0 - Utility for working with terraform based rpmbuilder farm

Copyright (C) 2006-2016 ESSENTIAL KAOS
Essential Kaos Open Source License <https://essentialkaos.com/ekol?en>
```

**Actual result**
```
Directory with farm data /opt/golang/src/github.com/essentialkaos/terrafarm/terradata/c6-multiarch is not exist
```

**Expected result**
```
-- TEMPLATES ---------------------------------------------------------------------------

  c6 (1 build node)
  foo (3 build node)
  bar (3 build node)

----------------------------------------------------------------------------------------
```

**Comment by andyone** [comment]
Thanks! It will be fixed soon.

**Comment by andyone** [comment]
Fixed in #18

**Comment by andyone** [comment]
@gongled Can you please check latest version?
